x/programs/runtime/import_program.go

```diff
@@ -15,6 +14,10 @@ const (
 	callProgramCost = 10000
 	setResultCost   = 10000
 	getFuelCost     = 10000
+
+	ExecutionFailure = 0
```

Comment:
Create an error type for these.

type ErrorCode uint8
ExecutionFailure ErrorCode = 0

Reply:
I have done it, let me know if that looks good. I think that the `[]byte{byte(ExecutionFailure)}` is looking pretty bad but your `AddProgram` branch might solve this.